manager/manager.h

```diff
@@ -48,6 +40,8 @@ class manager {
   YAML::Node config;
 
 /*
+ * Keeping this all here as we should start adding in functions here to do reading
```

Comment:
For T2K we really need save setting
https://github.com/t2k-software/MaCh3/blob/6b620ab3b38b843ac269f7ee7e9c7688f7adb6e6/manager/manager.cpp#L1730

Maybe not so huge but we need stuff like location of XsecMatrix, Osc matrix etc

Reply:
Yeah the manager class still needs a thing and how we handle "core" things. I imagine this will be done by passing the yaml file down to the base class as well in the derived class constructor and then the base class grabbing the information it needs.